index.html

```diff
@@ -0,0 +1,93 @@
+      <div class="burger-nav">
+        <nav id="nav-bar">
+        <div><a class="nav-link" href="#features">Features</a></div>
+        <div><a class="nav-link" href="#frame">How it works</a></div>
+        <div><a class="nav-link" href="#pricing">Pricing</a></div>
+      </nav></div>
+    </header>
+    
+    <div id="main">
+
+      <div id="form1">
+        <form id="form" action="https://www.freecodecamp.com/email-submit">
+        <label for="email" id="email-label"><b>Handcrafted, home-made masterpieces:</b></label><br><br>
+        <input type="email" placeholder="Enter your mail" id="email" name="email"  required><br><br>
+        <input type="submit" id="submit" value="GET STARTED">
+        </form>
+      </div><br><br><br>
+
+      <div id="features">
+        <div><img class="imag"  src="./img/icon1.jpg" alt="icon1"></div>
+        <div>
+          <b>Premium Materials</b><br>
+          Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase.
+        </div>
+      
+        <div><img class="imag"  src="./img/icon2.png" alt="icon2"></div>
+        <div>
+          <b>Fast Shipping</b><br>
+          We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns if you are not satisfied.
+        </div>
+      
+        <div><img class="imag" src="./img/icon3.jpg" alt="icon3"></div>
+        <div>
+          <b>Quality Assurance</b><br>
+          For every purchase you make, we will ensure there are no damages or faults and we will check and test the pitch of your instrument.
+        </div>
+         
+      </div>
+
+    <br><br><br><br>
+
+      <div id="frame">
+        <iframe id="video" src="https://www.youtube.com/embed/y8Yv4pnO7qc" title="YouTube video player" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>      
+      </div>
+    <br><br><br><br>
+
+      <div id="pricing">
+        <div id="item1" class="price"><h4>TENOR TROMBONE</h4>
+          <h2>$600</h2>
+          <p>Lorem ipsum.<br><br>
+          Lorem ipsum.<br><br>
+          Lorem ipsum dolor.<br><br>
+          Lorem ipsum.</p>
+          <form><input type="submit" class="submit" value="Select"></form></div>
+
+        <div id="item2" class="price"><h4>TENOR TROMBONE</h4>
+          <h2>$600</h2>
+          <p>orem ipsum.<br><br>
+          Lorem ipsum.<br><br>
+          Lorem ipsum dolor.<br><br>
+          Lorem ipsum.</p>
+          <form><input type="submit" class="submit" value="Select"></form></div>
+
+        <div id="item3" class="price"><h4>TENOR TROMBONE</h4>
+          <h2>$600</h2>
+          <p>Lorem ipsum.<br><br>
+          Lorem ipsum.<br><br>
+          Lorem ipsum dolor.<br><br>
+          Lorem ipsum</p>
+          <form><input type="submit" class="submit" value="Select"></form></div>
+      </div>
+
+    </div>
+  
+    
+    
```

Comment:
As an extra requirement, I would recommend adding a footer element.

Yout footer element should contain the following:
- Your social media links
- The webpage github issue link for reporting issues with the webpage
- The github repo link for showing the source code related with the current webpage.
See the below screenshot for more details.

![image](https://user-images.githubusercontent.com/36057474/146980022-009e4d54-09c6-4518-8ab7-f3e69c44e8ec.png)